stablehlo/reference/Element.cpp

```diff
@@ -350,6 +350,33 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+// These cases are implementation-defined, and the behavior of this interpreter:
+// For integers: if rhs is 0, the result is the same as `lhs`.
+//   For signed integers, if lhs is INT_MAX and rhs is -1, the result is 0.
+// For floats: use APFloat.mod(). If rhs is 0, the result is 0x7FF8000000000000
+Element rem(const Element &e1, const Element &e2) {
+  return map(
+      e1, e2,
+      [&](APInt lhs, APInt rhs) {
+        if (rhs == 0) return lhs;
+        return isSupportedSignedIntegerType(e1.getType()) ? lhs.srem(rhs)
+                                                          : lhs.urem(rhs);
+      },
+      [](bool lhs, bool rhs) -> bool {
+        llvm::report_fatal_error("reminder of two bool numbers is unsupported");
+      },
+      [](APFloat lhs, APFloat rhs) {
+        lhs.mod(rhs);
```

Comment:
Can we directly return the result, instead of having the return appear on the subsequent line?

Reply:
APFloat.mod() return status instead of a value, so current code can catch the error or return lhs. 
Please advise if there's a better way to do this?

Reply:
Oh I missed that. Let's leave a comment that we're deliberately ignoring this return value, and change this line to say `(void)lhs.mod(rhs)` to emphasize this.

Reply:
Yes.